docs/software/remote_desktop.md

```diff
@@ -180,6 +180,11 @@ XRDP is a remote desktop application using the *Windows Remote Desktop Client*.
 
     XRDP uses port **3389** by default, so you need to open/forward it from your router to DietPi.
 
+??? info "Troubleshooting: Microsoft Windows RDP connection black screen"
+
+    It is sometimes reported that a Windows RDP session hangs on a black screen when trying to connect to an XRDP server.  
+    This might be resolved using this instruction: [How to Resolve Microsoft RDP Connection Black Screen](https://techdirectarchive.com/2022/01/15/how-to-resolve-microsoft-rdp-connection-black-screen/) 
+
```

Comment:
The info bug is currently below the tabs. I suggest removing the tabs here all together. In all other cases, we have the info about which network port is used in the first/access tab. Aligning it here means that there is a single tab only, which doesn't make sense. All info here is immediately relevant when one wants to connect, so splitting it across tabs seems synthetically complicated 🤔.